.travis.yml

```diff
@@ -130,7 +130,6 @@ script:
           cd for_testing
           cp ../.coveragerc .
           nosetests --with-doctest --with-coverage --cover-package nibabel nibabel
-          pytest -v ../nibabel/tests/test_affines.py ../nibabel/tests/test_volumeutils.py
```

Comment:
Was masking nosetest failures. Removing for now.

Reply:
we also added to azure... https://github.com/nipy/nibabel/blob/master/.azure-pipelines/windows.yml#L44

Reply:
Amended. Thanks.